.editorconfig

```diff
@@ -0,0 +1,13 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
Sure - I do recommend it for open source projects though, to keep line endings and whitespace consistent :)